docs/optlib.rst

```diff
@@ -1573,7 +1573,7 @@ You can implement such roles by extending the built-in C parser:
 
 ctags with ``--options=c-extra.ctags --extras=+r --fields=+r`` emits
 
-.. code-block::
+.. code-block:: ctags
```

Comment:
This should be "tags" or specifying nothing.

Reply:
fixed.